sw/device/tests/alert_handler_lpg_clkoff_test.c

```diff
@@ -0,0 +1,619 @@
+        .alert_ids = kmac_alerts,
+        .num_alert_peri = num_kmac_alerts,
+        .clk_index = kTopEarlgreyHintableClocksMainKmac,
+        .is_hintable = true,
+    },
+    {
+        .name = "OTBN",
+        .base = TOP_EARLGREY_OTBN_BASE_ADDR,
+        .offset = OTBN_ALERT_TEST_REG_OFFSET,
+        .dif = &otbn,
+        .fatal_alert_bit = kDifOtbnAlertFatal,
+        .alert_ids = otbn_alerts,
+        .num_alert_peri = num_otbn_alerts,
+        .clk_index = kTopEarlgreyHintableClocksMainOtbn,
+        .is_hintable = true,
+    },
+    // Gateable clock IPs
+    {
+        .name = "SPI_HOST0",
+        .base = TOP_EARLGREY_SPI_HOST0_BASE_ADDR,
+        .offset = SPI_HOST_ALERT_TEST_REG_OFFSET,
+        .dif = &spi_host0,
+        .fatal_alert_bit = 0,
+        .alert_ids = spihost0_alerts,
+        .num_alert_peri = num_spihost0_alerts,
+        .clk_index = kTopEarlgreyGateableClocksIoPeri,
+        .is_hintable = false,
+    },
+    {
+        .name = "SPI_HOST1",
+        .base = TOP_EARLGREY_SPI_HOST1_BASE_ADDR,
+        .offset = SPI_HOST_ALERT_TEST_REG_OFFSET,
+        .dif = &spi_host1,
+        .fatal_alert_bit = 0,
+        .alert_ids = spihost1_alerts,
+        .num_alert_peri = num_spihost1_alerts,
+        .clk_index = kTopEarlgreyGateableClocksIoDiv2Peri,
+        .is_hintable = false,
+    },
+    {
+        .name = "USB",
+        .base = TOP_EARLGREY_USBDEV_BASE_ADDR,
+        .offset = USBDEV_ALERT_TEST_REG_OFFSET,
+        .dif = &usbdev,
+        .fatal_alert_bit = 0,
+        .alert_ids = usbdev_alerts,
+        .num_alert_peri = num_usbdev_alerts,
+        .clk_index = kTopEarlgreyGateableClocksUsbPeri,
+        .is_hintable = false,
+    },
+};
+
+/**
+ * Configure and enable alerts of `num_peripheral` peripherals starting from the
+ * `first_peripheral`
+ * @param num_peripherals The number of peripherals, of which alerts will be
+ * enabled
+ * @param first_peripheral The address of the first peripheral in the range
+ * @param ping_timeout The timeout value for the ping timer object.
+ */
+static void alert_handler_config_peripherals(uint32_t num_peripherals,
+                                             const test_t *first_peripheral,
+                                             uint32_t ping_timeout) {
+  dif_alert_handler_alert_t alerts[num_alerts];
+  dif_alert_handler_class_t alert_classes[num_alerts];
+
+  // Enable all alerts coming from the chosen peripherals
+  // Configure them as Class A.
+  size_t array_idx = 0;
+  test_t *peri_ptr;
+  for (int ii = 0; ii < num_peripherals; ii++) {
+    peri_ptr = (test_t *)first_peripheral + ii;
+    for (int jj = 0; jj < peri_ptr->num_alert_peri; jj++) {
+      alerts[array_idx] = peri_ptr->alert_ids[jj];
+      alert_classes[array_idx] = kDifAlertHandlerClassA;
+      // LOG_INFO("Periphereal name = %s, num_alert_peri = %d,
+      // alert_id=%d",peri_ptr->name, peri_ptr->num_alert_peri,
+      // peri_ptr->alert_ids[jj]);
```

Comment:
nit: remove commented-out code? Or if it's left in for debugging, then fix a typo:
```suggestion
      // LOG_INFO("Peripheral name = %s, num_alert_peri = %d,
      // alert_id=%d",peri_ptr->name, peri_ptr->num_alert_peri,
      // peri_ptr->alert_ids[jj]);
```

Reply:
Thanks, I removed all of the LOG_INFO statements. I was using them for debugging before.